src/pudl/transform/eia860m.py

```diff
@@ -69,8 +90,10 @@ def core_eia860m__changelog_generators(
     # there is one plant/gen that has duplicate values
     gens_idx = ["plant_id_eia", "generator_id", "report_date"]
     dupe_mask = (eia860m_all.plant_id_eia == 56032) & (eia860m_all.generator_id == "1")
-    deduped = eia860m_all[dupe_mask].drop_duplicates(subset=gens_idx, keep="first")
-    without_known_dupes = eia860m_all[~dupe_mask]
+    deduped: pd.DataFrame = eia860m_all[dupe_mask].drop_duplicates(
+        subset=gens_idx, keep="first"
+    )
+    without_known_dupes: pd.DataFrame = eia860m_all[~dupe_mask]
```

Comment:
I was getting some weird linting error about subsequent code being unreachable, which adding these type hints fixed. Not sure what was going on.

Reply:
hm that's really weird. I don't love that tbh

Reply:
Bug seems to have been fixed with some VS Code updates. Not sure what that was all about.